### DISCUSSION TITLE
Jean-Thierry Kéchichian Community Award 2024 - cast your votes!

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#16632] [General]
After the passing of our beloved [Jean-Thierry](https://docs.orchardcore.net/en/latest/community/jean-thierry/) to honor him, keep his memory alive, and foremost, inspire people to do good work for the community, we launched [an award named after him](https://docs.orchardcore.net/en/latest/community/jean-thierry-community-award/) for exceptional community members. Every year, the community votes on who has done the most for Orchard Core. After the [nominations](https://github.com/OrchardCMS/OrchardCore/discussions/16553), it's now time to vote!

**Cast your vote [here](https://www.rcv123.org/ballot/9f5JNN4axaQtoKcJHHmCya)!** Voting ends at 2:00 AM UTC on 12 September and we'll announce the winner at [Harvest](https://orchardcore.net/harvest)!

The vote uses ranked-choice voting provided by [RCV123.org](https://www.rcv123.org/). That way, you don't have to select a single community member but you can rank the nominees in your order of preference. In the end, we'll still have a single winner.

**What does the winner get?**

The award is foremost about recognition. However, the community provides some perks too:

- Mention of each year's winner [in the docs](https://docs.orchardcore.net/en/latest/community/jean-thierry-community-award/#winners-of-the-jean-thierry-kechichian-community-award).
- Post about it under the Orchard Core social media.
- [Lombiq](https://lombiq.com/) provides free custom domain usage on [DotNest](https://dotnest.com/), publicizing the winner in Lombiq's media, and a year of sponsorship similar to [other open source contributors and projects](https://lombiq.com/open-source).

If your company works with Orchard Core and would like to incentivize community members, please let us know what else you can add!

**Comment by hishamco** [comment]
@Piedone is there a way to protect submitting another vote from the same person

**Reply by Piedone** [reply]
There is some basic protection for that, yes (I'd assume with browser fingerprinting). Another would be to require people to provide their names, what I didn't want to.

**Comment by Piedone** [comment]
And the winner of Jean-Thierry Kéchichian Community Award 2024 is @hishamco, congratulations! I've sent you an e-mail about the perks :).
